src/number_theory/diophantine_approximation.lean

```diff
@@ -37,18 +47,27 @@ We also show a converse,
 Both statements are combined to give an equivalence,
 `real.infinite_rat_abs_sub_lt_one_div_denom_sq_iff_irrational`.
 
+There are two versions od Legendre's Theorem. One, `real.ex_rat_eq_convergent`, uses
```

Comment:
```suggestion
There are two versions of Legendre's Theorem. One, `real.ex_rat_eq_convergent`, uses
```

Reply:
I'll fix that (but I'll wait to see if there are further changes before bothering CI).